modules/olm-policy-scoping-operator-install.adoc

```diff
@@ -28,9 +31,15 @@ metadata:
   name: scoped
 EOF
 ----
+====
 
-. Allocate permissions that you want the Operator(s) to be confined to. This involves creating a new service account, relevant role(s), and role binding(s).
+. Allocate permissions that you want the Operator(s) to be confined to. This involves creating a new service account, relevant role(s), and role binding(s) in the newly created, designated namespace:
```

Comment:
[Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording): Use a colon at the end of the lead-in sentence unless another sentence intervenes between the lead-in sentence and the first list item. If a sentence intervenes, use a period to end both the lead-in sentence and the intervening sentence.

```suggestion
. Allocate permissions that you want the Operator(s) to be confined to. This involves creating a new service account, relevant role(s), and role binding(s) in the newly created, designated namespace.
```

Reply:
ISG says to use colons leading into substeps.